src/components/MediaRecord.tsx

```diff
@@ -225,7 +225,7 @@ function MediaRecord(props: IProps) {
   useEffect(() => {
     if (uploadSuccess !== undefined) {
       setUploading(false);
-      setFilechanged(!uploadSuccess);
+      setFilechanged(uploadSuccess);
```

Comment:
Most places that use MediaRecord expect it to be saved after the upload, so it should only be "savable" still if the upload failed.

Reply:
I tested this for ProvideRights and PassageDetailItem and both cases are fixed by this change.